app/views/core_induction_programme/modules/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">Content change preview</h1>
+    <%= form_with url: cip_year_module_path, method: :put do |f| %>
```

Comment:
Does this work without js? Do we care?

Reply:
Fascinatingly enough, when I disabled js, updating a module actually worked.

Preview didn't though :/ 

Might be because we are using a different route for preview (one that we defined ourselves), and Rails has some magic to handle default resource requests that fire off without js.

Reply:
Removing the concern in routes has done something so that preview now renders with JS disabled.